CHANGELOG.md

```diff
@@ -10,13 +10,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The report page now has a favicon [#223](https://github.com/scanapi/scanapi/pull/223)
 - Bandit security audit tool
 
-### Fixed
-- Updated language use in README.md and CONTRIBUTING.md plus fix broken links.
-- Removed unused sys import in scan.py and cleaned for PEP8 and spelling errors [#217](https://github.com/scanapi/scanapi/pull/217)
+## Changed
+- Renamed `api.(yaml|json)` to `scanapi.yaml` [#222](https://github.com/scanapi/scanapi/issues/20://github.com/scanapi/scanapi/pull/222)
 
 ### Removed
 - APIKeyMissingError
 
+### Fixed
```

Comment:
@supakeen Oh, I've just figured out that the changelog lint checks the sections by alphabetic order :woman_facepalming:  (Added, Changed, Fixed, ....)

But don't worry, for this time we can merge this way and I fix it later.

Reply:
👍